Automattic/wp-calypso
# i18n: Add `clean:translations` script

#### Changes proposed in this Pull Request

* Add `clean:translations` script that removes `calypso-strings.pot` and `chunks-map.[target].json` files.
* Call `clean:translations` script from parent `clean` script.

#### Testing instructions

* Build Calypso with `CALYPSO_ENV=production BUILD_TRANSLATION_CHUNKS=true yarn run build`
* Run `yarn run clean`
* Confirm that `calypso-strings.pot` and `chunks-map.[target].json` files have been removed from the root directory.
